packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -229,4 +229,31 @@ void main() {
 
     expect(find.text('Subtitle'), findsOneWidget);
   });
+
+  testWidgets('ExpansionTile padding test', (WidgetTester tester) async {
+    await tester.pumpWidget(const MaterialApp(
+      home: Material(
+        child: Center(
+          child: ExpansionTile(
+            title: Text('Hello'),
+            padding: EdgeInsets.fromLTRB(8, 12, 4, 10),
+          ),
+        ),
+      ),
+    ));
+
+    final Rect expansionTileRect = tester.getRect(find.byType(ExpansionTile));
+    final Rect titleRect = tester.getRect(find.text('Hello'));
+    final Rect trailingRect = tester.getRect(find.byIcon(Icons.expand_more));
+    final Rect tallerWidget = titleRect.height > trailingRect.height ? titleRect : trailingRect;
+
+    // Check the positions of title and trailing Widgets, after padding is applied.
+    expect(expansionTileRect.left, titleRect.left - 8);
+    expect(expansionTileRect.right, trailingRect.right + 4);
+
+    // Height of ListTile is 56.0, Border of ExpansionTile is 1.0, and top
```

Comment:
Could you just do a `tester.getRect(find.byType(ListTile))` to get the height of the ListTile here? That way, you can directly check to see if the `ListTile` itself has the correct padding without having to account for the border width and doing all the calculations in the current test

Reply:
Sure, I'll do the required changes.